tools/recipes/utils.sh

```diff
@@ -10,6 +10,7 @@ install_readme() {
     repo=$2
     licence_file="$repo/$3"
     src_info="$LISA_ARCH_ASSETS/README.$name"
+    musl_libc=$4
```

Comment:
As discussed, this can be replaced with an environment variable indicating an alpine build